libs/elasticsearch/langchain_elasticsearch/vectorstores.py

```diff
@@ -793,16 +458,30 @@ def similarity_search_with_score(
         Returns:
             List of Documents most similar to the query and score for each
         """
-        if isinstance(self.strategy, ApproxRetrievalStrategy) and self.strategy.hybrid:
+        if (
+            isinstance(self._store.retrieval_strategy, DenseVectorStrategy)
+            and self._store.retrieval_strategy.hybrid
+        ):
             raise ValueError("scores are currently not supported in hybrid mode")
 
-        return self._search(query=query, k=k, filter=filter, **kwargs)
+        hits = self._store.search(
+            query=query, k=k, filter=filter, custom_query=custom_query
+        )
+        return _hits_to_docs_scores(
+            hits=hits,
+            content_field=self.query_field,
+            doc_builder=doc_builder,
+        )
 
     def similarity_search_by_vector_with_relevance_scores(
         self,
         embedding: List[float],
         k: int = 4,
         filter: Optional[List[Dict]] = None,
+        custom_query: Optional[
```

Comment:
```suggestion
        *,
        custom_query: Optional[
```
Same here - make kwarg-only